src/Views/SuccessView.vala

```diff
@@ -42,9 +45,11 @@ public class SuccessView : AbstractInstallerView {
         bool requires_workaround = requires_workaround ();
 
         var description_label = new Gtk.Label (
-            requires_workaround
-                ? _("Shut down now and create a user account on the next start.")
-                : _("After restarting you can set up a new user, or you can shut down now and set up a new user later.")   
+            (this.mode == Modes.Mode.REFRESH)
+                ? _("System install has been refreshed. Restart now.")
+                : requires_workaround
+                    ? _("Shut down now and create a user account on the next start.")
```

Comment:
User already exists, so I don't think we should prompt about creating a user account here.